Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs

```diff
@@ -292,23 +292,29 @@ private async Task<(long Chunks, long Bytes)> StreamContentAsync(Stream reader,
             Debug.Assert(!(reader is null));
             Debug.Assert(!(responseStream is null));
 
+            int chunkSize = 0;
+            async Task<int> ReadNextChunk() { chunkSize = await reader.ReadAsync(buffer, 0, buffer.Length, ct); return chunkSize; }

```

Comment:
nit: could you please move it to the end of the method and switch to 'readNextChunk'? (this is the naming convention we use for local methods).